File: Basic Coding Problems/01 Fundamentals/010 Write a program to convert days into years, weeks and days/C++/days_to_years_weeks_days.cpp
#include <iostream>

using namespace std;

int main() {
    int number_of_days;
    int years;
    int weeks;
    int remaining_days;

    cout << "Enter the number of days: ";
    cin >> number_of_days;

    years = number_of_days / 365;
    remaining_days = number_of_days % 365;
    weeks = remaining_days / 7;
    remaining_days = remaining_days % 7;

    cout << "Years: " << years << endl;
    cout << "Weeks: " << weeks << endl;
    cout << "Days: " << remaining_days << endl;

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/010 Write a program to convert days into years, weeks and days/Java/days_to_years_weeks_days.java
import java.util.Scanner;

public class days_to_years_weeks_days {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the number of days: ");
        int number_of_days = scanner.nextInt();

        int years = number_of_days / 365;
        int remaining_days = number_of_days % 365;
        int weeks = remaining_days / 7;
        int remaining_days_2 = remaining_days % 7;

        System.out.println("Years: " + years);
        System.out.println("Weeks: " + weeks);
        System.out.println("Days: " + remaining_days_2);
    }
}